apps/dash-label-properties/app.py

```diff
@@ -0,0 +1,357 @@
+import dash
+from dash.dependencies import Input, Output, State
+import dash_html_components as html
+import dash_core_components as dcc
+import dash_bootstrap_components as dbc
+from dash.exceptions import PreventUpdate
+import dash_table
+from dash_table.Format import Format
+import plotly.express as px
+import plotly.graph_objs as go
+
+import numpy as np
+from skimage import io, filters, measure
+import pandas as pd
+
+import PIL
+from skimage import color, img_as_ubyte
+from plotly import colors
+from textwrap import dedent
+
+
+external_stylesheets = [dbc.themes.BOOTSTRAP]
+
+app = dash.Dash(__name__, external_stylesheets=external_stylesheets)
+server = app.server
+
+# Read the image that will be segmented
+filename = (
+    "https://upload.wikimedia.org/wikipedia/commons/a/ac/Monocyte_no_vacuoles.JPG"
+)
+img = io.imread(filename, as_gray=True)[:660:2, :800:2]
+labels = measure.label(img < filters.threshold_otsu(img))
+props = measure.regionprops(labels, img)
```

Comment:
here we can now use the `regionprops_table` function of scikit-image (recent addition), see for example https://scikit-image.org/docs/dev/auto_examples/segmentation/plot_regionprops_table.html#sphx-glr-auto-examples-segmentation-plot-regionprops-table-py The syntax will be more idiomatic for pandas users and so will (I think) be the creation of the `DataTable`